tools/cldr-code/src/test/java/org/unicode/cldr/api/CldrDataProcessorTest.java

```diff
@@ -0,0 +1,135 @@
+package org.unicode.cldr.api;
+
+import com.ibm.icu.dev.test.TestFmwk;
+import java.util.Arrays;
+import java.util.LinkedHashMap;
+import java.util.Map;
+import java.util.Objects;
+
+import com.google.common.collect.ImmutableMap;
+
+public class CldrDataProcessorTest extends TestFmwk {
```

Comment:
I don't understand why whether its copied affects whether we can/should use JUnit.

Reply:
It doesn't affect it. Just letting you know that JUnit is available, which is proably preferred for new tests. Since this is already present, there's no reason to rewrite it.